packages/frontend/editor-ui/src/components/VirtualSchema.test.ts

```diff
@@ -195,12 +195,12 @@ describe('VirtualSchema.vue', () => {
 		const { getAllByText, getAllByTestId } = renderComponent();
 
 		await waitFor(() =>
-			expect(getAllByText("No fields - item(s) exist, but they're empty").length).toBe(2),
+			expect(getAllByText('[Execute previous nodes] to see schema').length).toBe(2),
 		);
 
 		// Collapse second node
 		await userEvent.click(getAllByTestId('run-data-schema-header')[1]);
-		expect(getAllByText("No fields - item(s) exist, but they're empty").length).toBe(1);
+		expect(getAllByText('[Execute previous nodes] to see schema').length).toBe(1);
```

Comment:
It shouldn't always show this new text. When a node has not executed, it should show this. But if a node has executed and the output items are empty, it should show the current text.